services/office-add-in-api/lambdas/fetchReferenceCode.js

```diff
@@ -0,0 +1,75 @@
+import to from 'await-to-js';
+
+import config from '../../../config';
+import * as response from '../../../libs/response';
+import log from '../../../libs/logs';
+import * as dynamoDb from '../../../libs/dynamoDb';
+
+export const main = async (event, context) => {
```

Comment:
This should be implemented as an basic function to follow the rest of the code base. (As agreed in previous discussions a few months ago) :)